src/Testing.php

```diff
@@ -1,40 +0,0 @@
-<?php
-
-namespace Dxw\Iguana\Theme;
-
-trait Testing
```

Comment:
@william-man let's reword this commit message to say why we're deleting this (because we no longer use PHPUnit so it's not useful to us)